src/acl_kernel_if.cpp

```diff
@@ -90,8 +91,8 @@ acl_process_autorun_profiler_scan_chain(unsigned int physical_device_id,
 #define KERNEL_CSR_LMEM_INVALID_BANK 11
 #define KERNEL_CSR_LSU_ACTIVE 12
 #define KERNEL_CSR_WR_ACTIVE 13
-#define KERNEL_CSR_VALID_IN 14
-#define KERNEL_CSR_STARTED 15
+#define KERNEL_CSR_BUSY 14
+#define KERNEL_CSR_RUNNING 15
```

Comment:
Could you commit this as a separate change and motivate the renaming in the commit message?

Each runtime commit should ideally be atomic, i.e., a single, self-consistent change.

Reply:
Sure. I'll remove this renaming change from this patch.

Reply:
Thanks @shuoniu-intel! You can still have the renaming in this PR, just a separate commit. We use the rebase-merge strategy for this repository, so your multiple commits will be preserved as is on merge into `main`.

Reply:
Cool! Good to know that commits in a PR will not be combined after merging in this repository.